project-base/webpack.config.js

```diff
@@ -41,11 +42,12 @@ Encore
         beforeRun: () => {
             generateWebFont(
                 'frontend',
-                './assets/public/frontend/svg/*.svg'
+                './assets/public/frontend/svg/'
             );
             generateWebFont(
                 'admin',
-                './assets/public/admin/svg/*.svg'
+                sources.getFrameworkNodeModulesDir() + '/public/svg/',
```

Comment:
```suggestion
                sources.getFrameworkNodeModulesDir() + '/public/svg/admin/',
```

This is probably the reason why your build fails on Travis :wink:

Reply:
Hmm, I don't think so, because I used glob pattern `**/*.svg` and I moved all files into one-level folder `/web/public/admin/svg/`. My local build is OK, I have to think about it :smile:

Reply:
Ofcourse, async trable :smile: